tests/config_test.go

```diff
@@ -95,7 +96,7 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					"option2": "value2",
 					"option3": "value3",
 				}
-				tests.CreateConfigMap(configMapName, testsuite.GetTestNamespace(nil), data)
+				libconfigmap.New(configMapName, testsuite.GetTestNamespace(nil), data)
```

Comment:
The signature of libconfigmap.New is `func New(name string, data map[string]string) *kubev1.ConfigMap`
It does not take a namespace.

Reply:
Already fixed :)